icu4c/source/common/restrace.cpp

```diff
@@ -28,13 +28,33 @@ void ResourceTracer::trace(const char* resType) const {
     icu::CharString resPath;
     getResPath(resPath, status);
 
-    UTRACE_DATA3(UTRACE_VERBOSE, "(%s) %s @ %s",
+    // The longest type ("intvector") is 9 chars
+    icu::CharString format;
+    for (int32_t i=uprv_strlen(resType); i<9; i++) {
+        format.append(' ', status);
+    }
+    format.append("(%s) %s @ %s", status);
+
+    UTRACE_DATA3(UTRACE_VERBOSE,
+        format.data(),
         resType,
         filePath.data(),
         resPath.data());
     UTRACE_EXIT_STATUS(status);
 }
 
+void ResourceTracer::traceOpen() const {
+    U_ASSERT(fResB);
+    UTRACE_ENTRY(UTRACE_UDATA_BUNDLE);
+    UErrorCode status = U_ZERO_ERROR;
+
+    icu::CharString filePath;
+    getFilePath(filePath, status);
```

Comment:
Optional: If getFilePatch() returns a reference to the filePath, then you can use it inline, such as `UTRACE_DATAx(..., getFilePath(filePath, status).data());`